perf/istio-install/create_cluster.sh

```diff
@@ -141,7 +133,7 @@ function gc() {
 
   # shellcheck disable=SC2236
   set +u
```

Comment:
don't need this (and probably the shellcheck disable) if we are doing the `:-`

Reply:
Do you want me to submit a PR for this? If not, I'll clean it up next time I touch the file.